soh/soh/Enhancements/crowd-control/CrowdControl.cpp

```diff
@@ -101,7 +101,7 @@ void CrowdControl::Disable() {
 
 void CrowdControl::ListenToServer() {
     while (isEnabled) {
-        while (!connected) {
+        while (!connected && isEnabled) {
```

Comment:
why is this check being added here? shouldn't it be handled by the outer while loop?

Reply:
nevermind, i read this as `if` not `while`